auth_matrix/templates/admin/auth/user/change_list.html

```diff
@@ -0,0 +1,27 @@
+{% extends "admin/change_list.html" %}
+
+{% block extrastyle %}
+{{ block.super }}
+<style>
+    .extra {
+        padding: 1em;
+    }
+
+    .is-flex {
+        display: flex;
+    }
+</style>
```

Comment:
inline `<style>` is blegh, if you use the proper classes for the user action buttons that the django admin ships with, you should not need any additional CSS